hw/ip/flash_ctrl/data/flash_ctrl_testplan.hjson

```diff
@@ -24,199 +24,356 @@
       tests: ["flash_ctrl_smoke"]
     }
     {
-      name: backpressure
+      name: smoke_hw_rd
       desc: '''
-            Sanity + randomly pick between back-pressured reads / writes and fifo status-polled
-            reads / writes via the controller.
+            Perform host direct read on the single page of one bank within Data partition.
+            '''
+      milestone: V1
+      tests: []
+    }
+    {
+      name: sw_op_pwrmgr
+      desc: '''
+            Perform flash protocol controller read, program and erase on the single page of one
+            bank within Data partition. Finally perform read on same location in order to test
+            if previous operation was done successfully. Observe pwrmgr interface in order to
+            check status of operation.
+            '''
+      milestone: V2
+      tests: []
+    }    
+    {
+      name: host_read_direct
```

Comment:
i know we talked during the meeting about enabling backdoor load to check for scrambling correctness.
That can be probably be second priority at the moment, but probably still should document it.

Reply:
It is documented at testpoint "scramble".